docs/config/index.md

```diff
@@ -2330,6 +2331,13 @@ Annotation that is output at the end of diff result if it's truncated.
 
 Color of truncate annotation, default is output with no color.
 
+#### diff.printBasicPrototype
+
+- **Type**: `boolean`
+- **Default**: `true`
+
+Allows to set pretty-format option printBasicPrototype for diff output
```

Comment:
Isn't it already default `false` on Vitest? https://vitest.dev/guide/snapshot.html#_2-printbasicprototype-is-default-to-false

Reply:
Aha, I just notice these two `diff.printBasicPrototype` and `snapshotFormat.printBasicPrototype` are different things. I think adding this option is okay to start with, but I wonder if we should have these two same.

Btw, can you add a test?

Reply:
It should definitely be the same as the default now (disabled)

Reply:
take a look and tell me what will be the result of this evaluation?
https://github.com/vitest-dev/vitest/blob/main/packages/pretty-format/src/index.ts#L491

So I am not changing the default but adding missing possibility to set it from main config
If this should be default behavior (disabled)  the code I pointed must be changed

Reply:
> take a look and tell me what will be the result of this evaluation?

Adding tests will hopefully tell you that :smiley: 

Btw, changing the default will probably break a few our internal diff test. If you like, you can just add option in this PR and we can follow up changing the default separately.